index.js

```diff
@@ -233,7 +237,29 @@ module.exports = function (options) {
 
   router.get('/', function (req, res) {
     // If we've gotten this far then token gives proper access and that's all that this route care about
-    return res.sendStatus(200);
+    var isSyndicationRequest = (req.query.q === 'syndicate-to');
+    logger.debug({'query': req.query}, 'Received a query');
+
+    if (isSyndicationRequest) {
+      res.type('application/x-www-form-urlencoded');
+      var data = '';
+      for (var i = options.syndication.length - 1; i >= 0; i--) {
+        var syndicateTo = options.syndication[i];
+        data += 'syndicate-to[]=' + syndicateTo + '&';
+      }
+      res.format({
```

Comment:
While checking the `Accept` header and returning the appropriate data format is strictly speaking the correct thing to do I would lean more towards the [Robustness principle](https://en.wikipedia.org/wiki/Robustness_principle) and always send back form encoded data unless the client has specifically requested an alternative response format that's also supported (like if a JSON-based response format were supported).